src/main/java/vlad/petrovskyi/internetshop/controllers/shoppingcart/ShoppingCartController.java

```diff
@@ -0,0 +1,31 @@
+package vlad.petrovskyi.internetshop.controllers.shoppingcart;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import vlad.petrovskyi.internetshop.lib.Injector;
+import vlad.petrovskyi.internetshop.service.ShoppingCartService;
+
+public class ShoppingCartController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+
+    private static final Injector INJECTOR = Injector.getInstance("vlad.petrovskyi.internetshop");
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.setAttribute("products",
+                shoppingCartService.getAllProducts(shoppingCartService.getByUserId(USER_ID)));
+        request.setAttribute("sum", shoppingCartService.getAllProducts(
+                shoppingCartService.getByUserId(USER_ID))
```

Comment:
You are calling getByUserId() twice. Let's save the result of this method to local variable.